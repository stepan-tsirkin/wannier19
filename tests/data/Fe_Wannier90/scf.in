 &control
    calculation  = 'scf'
    restart_mode = 'from_scratch'
    pseudo_dir   = '.'
    outdir       = './temp'
    prefix       = 'Fe'
 /
 &system
    ibrav        = 3
    celldm(1)    = 5.4235
    nat          = 1
    ntyp         = 1
    ecutwfc      = 120.d0
    ecutrho      = 600.d0
    lspinorb     = .true.
    noncolin     = .true.
    starting_magnetization(1) = -1.d0
    occupations  = 'smearing'
    smearing     = 'm-v'
    degauss      = 0.02
 /
 &electrons
    startingpot  = 'file'
    conv_thr     = 1.0e-15
 /
ATOMIC_SPECIES
 Fe 55.85 Fe.dojo.fr.ONCVv0.4.pbe.standard.upf
ATOMIC_POSITIONS alat
 Fe 0.0 0.0 0.0
K_POINTS (automatic)
16 16 16 0 0 0
