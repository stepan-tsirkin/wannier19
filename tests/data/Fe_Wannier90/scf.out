
     Program PWSCF v.6.7GPU starts on 22Mar2021 at 13: 2:27 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on      16 processor cores
     Number of MPI processes:                16
     Threads/MPI process:                     1

     MPI processes distributed on     1 nodes
     K-points division:     npool     =       8
     R & G space division:  proc/nbgrp/npool/nimage =       2
     Reading input from scf.in

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3

     Fixed quantization axis for GGA:     0.000000    0.000000   -1.000000

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min         500     398    120                 9930     7085    1173
     Max         501     399    121                 9931     7086    1174
     Sum        1001     797    241                19861    14171    2347
 
     Using Slab Decomposition
 


     bravais-lattice index     =            3
     lattice parameter (alat)  =       5.4235  a.u.
     unit-cell volume          =      79.7644 (a.u.)^3
     number of atoms/cell      =            1
     number of atomic types    =            1
     number of electrons       =        16.00
     number of Kohn-Sham states=           24
     kinetic-energy cutoff     =     120.0000  Ry
     charge density cutoff     =     600.0000  Ry
     scf convergence threshold =      1.0E-15
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)
     Noncollinear calculation with spin-orbit


     celldm(1)=   5.423500  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   0.500000   0.500000   0.500000 )  
               a(2) = (  -0.500000   0.500000   0.500000 )  
               a(3) = (  -0.500000  -0.500000   0.500000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  1.000000 )  
               b(2) = ( -1.000000  1.000000  0.000000 )  
               b(3) = (  0.000000 -1.000000  1.000000 )  


     PseudoPot. # 1 for Fe read from file:
     ./Fe.dojo.fr.ONCVv0.4.pbe.standard.upf
     MD5 check sum: b9cfc4dc94fead0ebd2746b016929f5e
     Pseudo is Norm-conserving + core correction, Zval = 16.0
     Generated using ONCVPSP code by D. R. Hamann
     Using radial grid of 1426 points, 10 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   1
                l(6) =   1
                l(7) =   2
                l(8) =   2
                l(9) =   2
               l(10) =   2

     atomic species   valence    mass     pseudopotential
        Fe            16.00    55.85000     Fe( 1.00)

     16 Sym. Ops., with inversion, found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Fe  tau(   1) = (   0.0000000   0.0000000   0.0000000  )

     number of k points=   349  Marzari-Vanderbilt smearing, width (Ry)=  0.0200

     Number of k-points >= 100: set verbosity='high' to print them.

     Dense  grid:    19861 G-vectors     FFT dimensions: (  40,  40,  40)

     Smooth grid:    14171 G-vectors     FFT dimensions: (  36,  36,  36)

     Estimated max dynamical RAM per process >      56.56 MB

     Estimated total dynamical RAM >     904.91 MB
     Generating pointlists ...
     new r_m :   0.3572 (alat units)  1.9375 (a.u.) for type    1

     The initial density is read from file :
     ./temp/Fe.save/charge-density


 ==============================================================================
     atom number    1 relative position :    0.0000   0.0000   0.0000
     charge :    14.422172
     magnetization :         -0.000000    0.000000   -2.380880
     magnetization/charge:   -0.000000    0.000000   -0.165085
     polar coord.: r, theta, phi [deg] :     2.380880  180.000000  360.000000

 ==============================================================================
     Starting wfcs are   20 randomized atomic wfcs +    4 random wfcs

     total cpu time spent up to now is        1.9 secs

     Self-consistent Calculation

     iteration #  1     ecut=   120.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     c_bands:  1 eigenvalues not converged
     ethr =  1.00E-05,  avg # of iterations =  8.9

     Threshold (ethr) on eigenvalues was too large:
     Diagonalizing with lowered threshold

     Davidson diagonalization with overlap
     ethr =  1.49E-10,  avg # of iterations =  5.1

     total cpu time spent up to now is       20.3 secs

     total energy              =    -250.64467668 Ry
     estimated scf accuracy    <          3.7E-13 Ry

     total magnetization       =    -0.00    -0.00    -2.31 Bohr mag/cell
     absolute magnetization    =     2.47 Bohr mag/cell

     iteration #  2     ecut=   120.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-13,  avg # of iterations =  3.4

     total cpu time spent up to now is       26.5 secs

     total energy              =    -250.64467668 Ry
     estimated scf accuracy    <          1.7E-12 Ry

     total magnetization       =    -0.00     0.00    -2.31 Bohr mag/cell
     absolute magnetization    =     2.47 Bohr mag/cell

     iteration #  3     ecut=   120.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is       31.0 secs

     total energy              =    -250.64467668 Ry
     estimated scf accuracy    <          1.6E-13 Ry

     total magnetization       =     0.00    -0.00    -2.31 Bohr mag/cell
     absolute magnetization    =     2.47 Bohr mag/cell

     iteration #  4     ecut=   120.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is       35.4 secs

     total energy              =    -250.64467668 Ry
     estimated scf accuracy    <          6.1E-14 Ry

     total magnetization       =    -0.00     0.00    -2.31 Bohr mag/cell
     absolute magnetization    =     2.47 Bohr mag/cell

     iteration #  5     ecut=   120.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is       39.8 secs

     total energy              =    -250.64467668 Ry
     estimated scf accuracy    <          2.5E-14 Ry

     total magnetization       =    -0.00     0.00    -2.31 Bohr mag/cell
     absolute magnetization    =     2.47 Bohr mag/cell

     iteration #  6     ecut=   120.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is       44.2 secs

     total energy              =    -250.64467668 Ry
     estimated scf accuracy    <          1.2E-14 Ry

     total magnetization       =     0.00    -0.00    -2.31 Bohr mag/cell
     absolute magnetization    =     2.47 Bohr mag/cell

     iteration #  7     ecut=   120.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is       48.7 secs

     total energy              =    -250.64467668 Ry
     estimated scf accuracy    <          8.3E-15 Ry

     total magnetization       =    -0.00     0.00    -2.31 Bohr mag/cell
     absolute magnetization    =     2.47 Bohr mag/cell

     iteration #  8     ecut=   120.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is       53.1 secs

     total energy              =    -250.64467668 Ry
     estimated scf accuracy    <          6.3E-15 Ry

     total magnetization       =    -0.00    -0.00    -2.31 Bohr mag/cell
     absolute magnetization    =     2.47 Bohr mag/cell

     iteration #  9     ecut=   120.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is       57.5 secs

     total energy              =    -250.64467668 Ry
     estimated scf accuracy    <          4.8E-15 Ry

     total magnetization       =    -0.00     0.00    -2.31 Bohr mag/cell
     absolute magnetization    =     2.47 Bohr mag/cell

     iteration # 10     ecut=   120.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-13,  avg # of iterations =  1.0

     total cpu time spent up to now is       61.9 secs

     total energy              =    -250.64467668 Ry
     estimated scf accuracy    <          3.0E-15 Ry

     total magnetization       =    -0.00     0.00    -2.31 Bohr mag/cell
     absolute magnetization    =     2.47 Bohr mag/cell

     iteration # 11     ecut=   120.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-13,  avg # of iterations =  1.0

 ==============================================================================
     atom number    1 relative position :    0.0000   0.0000   0.0000
     charge :    14.422172
     magnetization :          0.000000   -0.000000   -2.380880
     magnetization/charge:    0.000000   -0.000000   -0.165085
     polar coord.: r, theta, phi [deg] :     2.380880  180.000000  360.000000

 ==============================================================================

     total cpu time spent up to now is       66.3 secs

     End of self-consistent calculation

     Number of k-points >= 100: set verbosity='high' to print the bands.

     the Fermi energy is    17.6255 ev

!    total energy              =    -250.64467668 Ry
     estimated scf accuracy    <          8.1E-19 Ry
     smearing contrib. (-TS)   =       0.00093279 Ry
     internal energy E=F+TS    =    -250.64560948 Ry

     The total energy is F=E-TS. E is the sum of the following terms:
     one-electron contribution =    -105.84231181 Ry
     hartree contribution      =      62.52454367 Ry
     xc contribution           =     -35.54877559 Ry
     ewald contribution        =    -171.77906575 Ry

     total magnetization       =     0.00    -0.00    -2.31 Bohr mag/cell
     absolute magnetization    =     2.47 Bohr mag/cell

     convergence has been achieved in  11 iterations

     Writing output data file ./temp/Fe.save/
 
     init_run     :      1.71s CPU      1.74s WALL (       1 calls)
     electrons    :     62.57s CPU     64.46s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      1.49s CPU      1.50s WALL (       1 calls)
     potinit      :      0.07s CPU      0.08s WALL (       1 calls)
     hinit0       :      0.09s CPU      0.09s WALL (       1 calls)

     Called by electrons:
     c_bands      :     52.26s CPU     53.97s WALL (      12 calls)
     sum_band     :      9.28s CPU      9.36s WALL (      12 calls)
     v_of_rho     :      0.70s CPU      0.71s WALL (      12 calls)
     mix_rho      :      0.22s CPU      0.24s WALL (      12 calls)

     Called by c_bands:
     init_us_2    :      0.46s CPU      0.61s WALL (    1100 calls)
     cegterg      :     45.90s CPU     46.29s WALL (     528 calls)

     Called by *egterg:
     cdiaghg      :      1.20s CPU      1.20s WALL (    1729 calls)
     h_psi        :     40.57s CPU     40.91s WALL (    1817 calls)
     g_psi        :      0.28s CPU      0.29s WALL (    1245 calls)

     Called by h_psi:
     h_psi:calbec :      1.44s CPU      1.43s WALL (    1817 calls)
     vloc_psi     :     37.58s CPU     37.95s WALL (    1817 calls)
     add_vuspsi   :      1.15s CPU      1.13s WALL (    1817 calls)

     General routines
     calbec       :      1.44s CPU      1.42s WALL (    1817 calls)
     fft          :      0.21s CPU      0.21s WALL (     386 calls)
     ffts         :      0.04s CPU      0.04s WALL (      96 calls)
     fftw         :     36.12s CPU     36.36s WALL (  155116 calls)
     interpolate  :      0.05s CPU      0.05s WALL (      48 calls)
 
     Parallel routines
 
     PWSCF        :   1m 6.65s CPU   1m14.93s WALL

 
   This run was terminated on:  13: 3:42  22Mar2021            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
